react_app/src/components/Input/Input.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Input from "./Input.jsx";
+
+export default {
+  title: "Input",
+  component: Input,
+};
+
+export const InputStories = () => (
+  <Input
+    name='add text'
+    title='Please add text'
+    type='text'
+    value='add text'
+    onChange={() => {}}
+    defaultValue='add text'
+  />
+);
```

Comment:
You can add `Controls` into storybook UI by doing this:
```jsx
const Template = (args) => <Input {...args} />;

export const Default = Template.bind({});

Default.args = {
  name:'add text',
  title:'Please add text',
  type:'text',
  value:'add text',
  onChange:() => {},
  defaultValue:'add text',
};
```
<details>
<summary>Storybook UI</summary>

![image](https://user-images.githubusercontent.com/73696363/229853275-54cdabc3-90c0-4499-8fe3-6d59d4f38576.png)
</details>